src/registry/registry.ts

```diff
@@ -32,13 +40,18 @@ export interface FieldSubmissionHandlerRegistration extends ComponentRegistratio
   type: string;
 }
 
+export interface FormScemaTransformerRegistration extends ComponentRegistration<FormSchemaTransformer> {
+  type: string;
+}
```

Comment:
There are two issues:

1. There is a typo in the interface name
2. Do we need the `type` property?